README.md

```diff
@@ -326,26 +326,44 @@ Additionally, users can utilize the MiNiFi Toolkit Converter (version 0.0.1 - sc
 
 ### Http Configuration Listener
 
-    Http Configuration Listener will pull flow file configuration from the remote command control server, 
+    Http Configuration Listener will pull flow file configuration from the remote command control server,
     validate and apply the new flow configuration
 
-    in minifi.properties 
+    in minifi.properties
 
     nifi.configuration.listener.type=http
     nifi.configuration.listener.http.url=https://localhost:8080
-    nifi.configuration.listener.pull.interval=1 sec 
+    nifi.configuration.listener.pull.interval=1 sec
     nifi.configuration.listener.client.ca.certificate=./conf/nifi-cert.pem
 
     if you want to enable client certificate
     nifi.configuration.listener.need.ClientAuth=true
     nifi.configuration.listener.client.certificate=./conf/client.pem
     nifi.configuration.listener.client.private.key=./conf/client.key
     nifi.configuration.listener.client.pass.phrase=./conf/password
-      
+
+### UID generation
+
+MiNiFi need to generate many unique identifiers in the course of operations.  There are a few different uid implementations available that can be configured in minifi-uid.properties.
+
+Implementation for uid generation can be selected using the uid.implementation property values:
+1. time - use uuid_generate_time
+2. random - use uuid_generate_random
+3. uuid_default - use uuid_generate (will attempt to use uuid_generate_random and fall back to uuid_generate_time if no high quality randomness is available)
```

Comment:
I added a comment in Id.cpp to clarify the location in code I was referring to. Sorry for the confusion.